.eslintrc.cjs

```diff
@@ -37,18 +37,11 @@ module.exports = {
         'hive-focus',
         'hive-focus-within',
         'nextra-hamburger',
-        '@container', // Tailwind ESLint Plugin doesn't see the Container Queries classes but it does see prefixes like @sm:
+        '@container', // Tailwind ESLint Plugin doesn't see the Container Queries classes, but it does see prefixes like @sm:
```

Comment:
do u have sth like Grammarly running in your editor? I wanna get back to writing blog posts

Reply:
yeah, I have some extension in my WebStorm, if you want a name I can take a look